File: fizzbuzz.php
<?php
function fizzbuzz ($number) {
  if ($number % 15 == 0 ) {
      echo "fizzbuzz";
  } elseif ($number % 3 == 0 ) {
      echo "fizz";
  } elseif ($number % 5 == 0 ) {
      echo "buzz";
  } else {
      echo $number;
  }
}

$counter = 1;

while ($counter <= 100) {
    fizzbuzz($counter);
	echo "\n";
    $counter++;
}
?>

File: fizzbuzz.pl
sub fizzbuzz {
    if ( $_[0] % 15 == 0 ) {
        print "fizzbuzz\n";
    } elsif ( $_[0] % 3 == 0 ) {
        print "fizz\n";
    } elsif ( $_[0] % 5 == 0 ) {
        print "buzz\n";
    } else {
        print $_[0];
        print "\n";
    }
}

$counter = 1;

while ( $counter < 101 ) {
    fizzbuzz($counter)
    $counter = $counter + 1;
}
